docs/nRF_Cloud_Location_services/Configure_nRF_Cloud_integration.md

```diff
@@ -0,0 +1,36 @@
+# Connect nRF Cloud integration
+
+Connect your nRF Cloud devices to Coiote DM to track their location and save battery life. [Read more about Nordic Semiconductor Location services](https://www.nordicsemi.com/Products/Cloud-services?lang=en#infotabs).
+
+## Prerequisites
+
+  - An nRF Cloud account. If you don't have it, [create one here](https://nrfcloud.com/#/).
+  - A Coiote DM user account with permissions to use the nRF Cloud integration.
```

Comment:
Every user can try Locations services. This feature is available in the Developer plan for free. 
In this case, maybe we can remove "with permissions"?

Reply:
If this permission is granted by default for every user who registers in eu.cloud, then this mention can be removed

Reply:
looks good